chart/openfaas/values.yaml

```diff
@@ -15,13 +15,19 @@ gateway:
   readTimeout : "20s"
   writeTimeout : "20s"
   upstreamTimeout : "15s"  # Must be smaller than read/write_timeout
+  replicas: 1
 
 queueWorker:
   ackWait : "30s"
+  replicas: 1
 
 # image pull policy for openfaas components, can change to `IfNotPresent` in offline env
 openfaasImagePullPolicy: "Always"
```

Comment:
While we are making so many other unrelated changes perhaps this should also set the pull policy for the system components such as the queue worker and gateway via a new option. See the open issues for why this is important for our users.